src/main/java/com/example/demo/services/impl/UserServiceImpl.java

```diff
@@ -19,27 +20,29 @@
 public class UserServiceImpl implements UserService {
 
     private final UserRepository userRepository;
+
     private final UserMapper userMapper;
 
     private final UserEntityMapper userEntityMapper;
-    public UserServiceImpl(UserRepository userRepository, UserMapper userMapper, UserEntityMapper userEntityMapper) {
+
+    public UserServiceImpl(final UserRepository userRepository, final UserMapper userMapper, final UserEntityMapper userEntityMapper) {
         this.userRepository = userRepository;
         this.userMapper = userMapper;
         this.userEntityMapper = userEntityMapper;
     }
 
-
     @Override
     public List<User> getUsers() {
-        List<UserEntity> userEntities = userRepository.findAll();
-        List<User> users = userMapper.map(userEntities);
-        return users;
+        return userRepository
+                .findAll()
+                .stream()
+                .map(userEntity -> userMapper.map(userEntity))
+                .toList();
```

Comment:
Methods chaining looks nice. Good job.
This is what we need to achieve each time.